dll/win32/shimgvw/shimgvw.c

```diff
@@ -241,9 +242,41 @@ BOOL Anime_Step(DWORD *pdwDelay)
     return FALSE;
 }
 
+static void UpdateZoom(UINT NewZoom)
+{
+    BOOL bEnableZoomIn, bEnableZoomOut;
+
+    /* If zoom has not been changed, ignore it */
+    if (ZoomPercents == NewZoom)
+        return;
+
+    ZoomPercents = NewZoom;
+
+    /* Check if a zoom button of the toolbar must be grayed */
+    bEnableZoomIn = TRUE;
+    bEnableZoomOut = TRUE;
+
+    if (NewZoom >= MAX_ZOOM)
+    {
+        bEnableZoomIn = FALSE;
+    }
+    else
+    if (NewZoom <= MIN_ZOOM)
```

Comment:
```suggestion
    else if (NewZoom <= MIN_ZOOM)
```

Reply:
I could do it, but this is against the coding rules ("Each statement should get its own line.").

Reply:
Coding Style wiki page does not cover `else if` part, however... https://git.reactos.org/?p=reactos.git&a=search&h=HEAD&st=grep&s=else+if

Reply:
besides, `else if` is considered as 1 statement.